--- Database.cpp	2010-11-18 16:58:02.000000000 -0700
+++ ../~Docugate (DGC App Release 7.3)/Database.cpp	2011-12-15 11:23:04.000000000 -0700
@@ -80,6 +80,10 @@
 
 						if( varInfo.m_strName.CompareNoCase("CTLUNIT_GUID") == 0 )
 					         strncpy(Stations[StationCount].UnitGUID,varData,51);
+						
+						// Added by Mike on 12/14/11
+						if( varInfo.m_strName.CompareNoCase("CTLUNIT_OU_GUID") == 0 )
+					         strncpy(Stations[StationCount].UnitOUGUID,varData,51); 
 
 						if( varInfo.m_strName.CompareNoCase("CTLUNIT_IP") == 0 )
 					         strncpy(Stations[StationCount].UnitAddress,varData,50);
@@ -584,19 +588,37 @@
 		m_Query.Format("SELECT ACC_DEFAULT_CLIENT,ACC_DEFAULT_MATTER,ACC_LOCKED,ACC_FUNDS,ACC_OU_GUID,ACC_CMETHOD FROM DOCUGATE_GENERAL_ACC WHERE ACC_ACC_GUID = '%s'", McrStation.AccountGUID);
 		m_RS.Open(AFX_DB_USE_DEFAULT_TYPE, m_Query,CRecordset::executeDirect);
 
+		/*
 		if ( !m_RS.IsEOF() )
-		{
-			CString Data;
-			m_RS.GetFieldValue( short(0), Data); lstrcpyn(McrStation.AccountClient,Data,50);
-			m_RS.GetFieldValue( short(1), Data); lstrcpyn(McrStation.AccountMatter,Data,50);
-			m_RS.GetFieldValue( short(2), Data); McrStation.AccountLocked  = atoi(Data);
-			m_RS.GetFieldValue( short(3), Data); McrStation.InitBalance = (float)atof(Data);
-			m_RS.GetFieldValue( short(4), Data); lstrcpyn(McrStation.Group,Data,50);
-			m_RS.GetFieldValue( short(5), Data); lstrcpyn(McrStation.AccountCharge,Data,50);
-		}
+			// ACL comparision added by Mike on 12/14/11
+			if ( &McrStation.UnitOUGUID > 0 && strcmp(McrStation.Group,McrStation.UnitOUGUID) != 0 )
+				m_RS.Close();
+			else
+			{
+				CString Data;
+				m_RS.GetFieldValue( short(0), Data); lstrcpyn(McrStation.AccountClient,Data,50);
+				m_RS.GetFieldValue( short(1), Data); lstrcpyn(McrStation.AccountMatter,Data,50);
+				m_RS.GetFieldValue( short(2), Data); McrStation.AccountLocked  = atoi(Data);
+				m_RS.GetFieldValue( short(3), Data); McrStation.InitBalance = (float)atof(Data);
+				m_RS.GetFieldValue( short(4), Data); lstrcpyn(McrStation.Group,Data,50);
+				m_RS.GetFieldValue( short(5), Data); lstrcpyn(McrStation.AccountCharge,Data,50);
+			}
+		*/
+		if ( !m_RS.IsEOF() )
+			
+			{
+				CString Data;
+				m_RS.GetFieldValue( short(0), Data); lstrcpyn(McrStation.AccountClient,Data,50);
+				m_RS.GetFieldValue( short(1), Data); lstrcpyn(McrStation.AccountMatter,Data,50);
+				m_RS.GetFieldValue( short(2), Data); McrStation.AccountLocked  = atoi(Data);
+				m_RS.GetFieldValue( short(3), Data); McrStation.InitBalance = (float)atof(Data);
+				m_RS.GetFieldValue( short(4), Data); lstrcpyn(McrStation.Group,Data,50);
+				m_RS.GetFieldValue( short(5), Data); lstrcpyn(McrStation.AccountCharge,Data,50);
+			}
 	
 		if ( m_RS.IsOpen() )
-			 m_RS.Close();
+				m_RS.Close();
+		
 	}
 	catch( CDBException *e)
 	{
@@ -829,7 +851,8 @@
 		else
 			m_Query.Format("SELECT ACC_ACC_GUID FROM DOCUGATE_GENERAL_ACC WHERE ACC_COPYID = '%s'", McrStation.CopyAccount);
 	
-		
+		m_Query.Format("SELECT ACC_OU_GUID FROM DOCUGATE_GENERAL_ACC WHERE ACC_ACC_GUID = '%s'", McrStation.AccountGUID); //Added by Mike 12/14/11
+
 		m_RS.Open(AFX_DB_USE_DEFAULT_TYPE, m_Query,CRecordset::executeDirect);
 		
 		if ( !m_RS.IsEOF() )
@@ -841,6 +864,7 @@
 
 		if ( m_RS.IsOpen() )
 			 m_RS.Close();
+		
 
 	}
 	catch( CDBException *e)
